otter/controller.py

```diff
@@ -268,19 +277,18 @@ def check_cooldowns(log, state, config, policy, policy_id):
     return True
 
 
-def calculate_delta(log, state, config, policy):
+def apply_delta(log, current, state, config, policy):
     """
-    Calculate the desired change in the number of servers, keeping in mind the
-    minimum and maximum constraints.
+    Calculate a new desired number of servers based on a policy, assign that
```

Comment:
"Calculate a new desired number of servers based on a policy _on current_"? Also, will be nice to document `current`.